uvtools/tests/test_dspec.py

```diff
@@ -434,6 +451,48 @@ def get_snr(clean, fftax=1, avgax=0, modes=[2, 20]):
     rfft2 = np.fft.ifft2(res2)
     nt.assert_true(np.median(np.abs(rfft2[:15, :23] / rfft[:15, :23])) < 1)
 
+def test_delay_interpolation_matrix():
+    """
+    Test inerpolation matrix.
```

Comment:
Great docstring, thanks!

Reply:
Thanks for pointing out where the improvement was needed ;)